app/config/FrontendAppConfig.scala

```diff
@@ -64,6 +64,8 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
 
   lazy val sendEmailUrl: String = configuration.get[Service]("microservice.services.email").baseUrl
 
+  lazy val emailFailureAlertMessage = "The email could not be sent to the EMAIL service"
```

Comment:
@ers2015112 Let's log it as a string, no need to create it as constant AppConfig